src/Tools/Directory.Build.targets

```diff
@@ -0,0 +1,37 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.targets"/>
+
+  <Target Name="_GetPackageVersionInfo" DependsOnTargets="$(GetPackageVersionDependsOn)"
+          Returns="@(_ProjectPathWithVersion)">
+    <ItemGroup>
+      <_ProjectPathWithVersion Include="$(MSBuildProjectFullPath)">
+        <PackageVersion>$(Version)</PackageVersion>
+      </_ProjectPathWithVersion>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CollectBundleFilesToSign"
+          DependsOnTargets="ComputeFilesToPublish"
+          AfterTargets="ComputeFilesToPublish"
+          Returns="@(FilesInBundleToSign)">
+    <ItemGroup>
+      <FilesInBundleToSign Include="@(_FilesToBundle)" />
```

Comment:
We should add a comment mentioning that we get this item group from the internals of the Single-File publishing tasks.